src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super(color);
+        this.radius = radius;
+        super.color = color;
+    }
```

Comment:
```suggestion
    public Circle(double radius, String color) {
        super(color);
        this.radius = radius;
    }
```
same for all figures
color is set because you call super-class constructor